packages/augur-ui/src/modules/universe-cards/components/universe-card.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {
+  Breakdown,
+  ButtonsRow,
+} from 'modules/modal/common';
+import { DefaultButtonProps } from 'modules/common/buttons';
+import {
+  LinearPropertyLabelProps,
+} from 'modules/common/labels';
+
+import Styles from 'modules/universe-cards/universe-card.styles.less';
+
+interface UniverseCardProps {
+  universeId: string;
+  creationTimestamp: string;
+  outcomeName: string;
+  currentUniverse: string;
+  buttons: DefaultButtonProps[];
+  breakdown?: LinearPropertyLabelProps[];
+}
+
+export const UniverseCard = ({
+  universeId,
+  creationTimestamp,
+  outcomeName,
+  currentUniverse,
+  breakdown,
+  buttons
+}: UniverseCardProps) => {
+
+  return (
+    <div className={Styles.UniverseCard}>
+      <div>
+        {universeId === currentUniverse &&
+          <span>Current Universe</span>
+        }
+        <div>
+          <h1>Universe</h1>
+          <div className={Styles.outcomeName}>{outcomeName}</div>
```

Comment:
you don't need to have class names for this and the creationTimestamp one

Reply:
Oops, yeah I was originally using those, but then made some changes to minimize the use of class names and forgot to remove those from this file.